app/api/dao/mentorship_relation.py

```diff
@@ -290,6 +290,13 @@ def cancel_relation(user_id, relation_id):
 
         # All was checked
         request.state = MentorshipRelationState.CANCELLED
+
+        # Updates cancellation reason if it was passed in arguments
+        try:
+            request.cancellation_reason = data["cancellation_reason"]
+        except:
```

Comment:
Catch the exact exceptions you're looking to handle rather than simply catching everything and assigning None